policyengine_us/variables/gov/states/ri/tax/income/credits/property_tax/ri_property_tax_credit.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class ri_property_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Rhode Island property tax credit"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "http://webserver.rilin.state.ri.us/Statutes/TITLE44/44-33/44-33-9.htm"
+    )
+    defined_for = StateCode.RI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.ri.tax.income.credits.property_tax
+        agi = tax_unit("adjusted_gross_income", period)
```

Comment:
They use household income, which they define on page 2 of the form, and which is more expansive than AGI.